packages/codegen/src/codegen/.hygen/templates/powerhouse/generate-document-model-subgraph/index.js

```diff
@@ -1,13 +1,25 @@
 // @ts-check
-const { paramCase } = require("change-case");
+const { paramCase, pascalCase, camelCase } = require("change-case");
 
-const generateDocumentModelMutations = {
+module.exports = {
   params: ({ args }) => {
     const documentModel = JSON.parse(args.documentModel);
     const latestSpec =
       documentModel.specifications[documentModel.specifications.length - 1];
-
+    const documentType = documentModel.name;
```

Comment:
We can make our lives much easier and save a ton of redundant calls to h.changeCase all over the place by just doing it once and assigning to a variable which we then use in the template